<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAKDebugPage</class>
 <widget class="QWidget" name="SAKDebugPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="verticalSpacing">
    <number>4</number>
   </property>
   <item row="6" column="11" colspan="2">
    <widget class="QCheckBox" name="showDateCheckBox">
     <property name="text">
      <string>输出日期</string>
     </property>
    </widget>
   </item>
   <item row="3" column="13" colspan="2">
    <widget class="QLabel" name="rxLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
    color:red
}</string>
     </property>
     <property name="text">
      <string notr="true">Rx&lt;&lt;&lt;&lt;&lt;&lt;&lt;&lt;&lt;&lt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="13" column="13" colspan="2">
    <widget class="QLabel" name="txSpeedLabel">
     <property name="styleSheet">
      <string notr="true">QLabel{
    color:blue
}</string>
     </property>
     <property name="text">
      <string notr="true">0B/s</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="30" column="1">
    <widget class="QComboBox" name="inputModelComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="14" column="13" colspan="2">
    <widget class="QLabel" name="txFramesLabel">
     <property name="styleSheet">
      <string notr="true">QLabel{
    color:blue
}</string>
     </property>
     <property name="text">
      <string notr="true">0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="17" column="11" colspan="2">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>接收帧数</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="31" column="1">
    <widget class="QPushButton" name="saveInputDataPushButton">
     <property name="text">
      <string>保存输入</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="styleSheet">
      <string notr="true">QLabel{
    color:green
}</string>
     </property>
     <property name="text">
      <string>设备控制</string>
     </property>
    </widget>
   </item>
   <item row="5" column="13" colspan="2">
    <widget class="QComboBox" name="outputTextFormatComboBox"/>
   </item>
   <item row="27" column="3" colspan="6">
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="19" column="11" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="30" column="0">
    <widget class="QLabel" name="label_23">
     <property name="text">
      <string>输入数据格式</string>
     </property>
    </widget>
   </item>
   <item row="20" column="11" colspan="2">
    <widget class="QPushButton" name="resetTxCountPushButton">
     <property name="text">
      <string>复位发送</string>
     </property>
    </widget>
   </item>
   <item row="29" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>CRC参数模型</string>
     </property>
    </widget>
   </item>
   <item row="27" column="0" colspan="2">
    <widget class="QLabel" name="label_28">
     <property name="styleSheet">
      <string notr="true">QLabel{
    color:green
}</string>
     </property>
     <property name="text">
      <string>输入设置</string>
     </property>
    </widget>
   </item>
   <item row="28" column="11" colspan="2">
    <widget class="QPushButton" name="transmissionSettingPushButton">
     <property name="text">
      <string>数据转发</string>
     </property>
    </widget>
   </item>
   <item row="13" column="11" colspan="2">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>发送速率</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="11" colspan="2">
    <widget class="QCheckBox" name="showTimeCheckBox">
     <property name="text">
      <string>输出时间</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="28" column="13" colspan="2">
    <widget class="QPushButton" name="readWriteSettingPushButton">
     <property name="text">
      <string>读写参数</string>
     </property>
    </widget>
   </item>
   <item row="18" column="11" colspan="2">
    <widget class="QLabel" name="label_18">
     <property name="text">
      <string>接收字节</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="31" column="2">
    <widget class="QPushButton" name="clearInputPushButton">
     <property name="text">
      <string>清空输入</string>
     </property>
    </widget>
   </item>
   <item row="1" column="11" colspan="4">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="13" colspan="2">
    <widget class="QPushButton" name="saveOutputPushButton">
     <property name="text">
      <string>保存输出</string>
     </property>
    </widget>
   </item>
   <item row="27" column="10">
    <widget class="QLabel" name="crcLabel">
     <property name="styleSheet">
      <string notr="true">QLabel{
    color:blue
}</string>
     </property>
     <property name="text">
      <string notr="true">0xFFFFFFFF</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="27" column="11" colspan="4">
    <widget class="QLabel" name="label_9">
     <property name="styleSheet">
      <string notr="true">QLabel{
    color:green
}</string>
     </property>
     <property name="text">
      <string>其他设置</string>
     </property>
    </widget>
   </item>
   <item row="15" column="11" colspan="2">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>发送字节</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="15" column="13" colspan="2">
    <widget class="QLabel" name="txBytesLabel">
     <property name="styleSheet">
      <string notr="true">QLabel{
    color:blue
}</string>
     </property>
     <property name="text">
      <string notr="true">0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="14" column="11" colspan="2">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>发送帧数</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="31" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="27" column="2">
    <widget class="QLabel" name="label_26">
     <property name="styleSheet">
      <string notr="true">QLabel{
    color:green
}</string>
     </property>
     <property name="text">
      <string>数据输入</string>
     </property>
    </widget>
   </item>
   <item row="20" column="13" colspan="2">
    <widget class="QPushButton" name="resetRxCountPushButton">
     <property name="text">
      <string>复位接收</string>
     </property>
    </widget>
   </item>
   <item row="2" column="11" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>发送状态</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="31" column="9">
    <widget class="QPushButton" name="sendPresetPushButton">
     <property name="text">
      <string>发送预设</string>
     </property>
    </widget>
   </item>
   <item row="31" column="8">
    <widget class="QPushButton" name="presetPushButton">
     <property name="text">
      <string>预设数据</string>
     </property>
    </widget>
   </item>
   <item row="30" column="11" colspan="2">
    <widget class="QPushButton" name="timingSendingPushButton">
     <property name="text">
      <string>定时发送</string>
     </property>
    </widget>
   </item>
   <item row="31" column="5">
    <widget class="QCheckBox" name="addCRCCheckBox">
     <property name="text">
      <string>添加CRC</string>
     </property>
    </widget>
   </item>
   <item row="16" column="11" colspan="2">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>接收速率</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="28" column="1">
    <widget class="QLineEdit" name="cycleTimeLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="11" colspan="2">
    <widget class="QLabel" name="txLabel">
     <property name="styleSheet">
      <string notr="true">QLabel{
    color:blue
}</string>
     </property>
     <property name="text">
      <string notr="true">Tx&gt;&gt;&gt;&gt;&gt;&gt;&gt;&gt;&gt;&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="31" column="4">
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="11" column="11" colspan="4">
    <widget class="QLabel" name="label_10">
     <property name="styleSheet">
      <string notr="true">QLabel{
    color:green
}</string>
     </property>
     <property name="text">
      <string>数据统计</string>
     </property>
    </widget>
   </item>
   <item row="8" column="11" colspan="2">
    <widget class="QCheckBox" name="showRxDataCheckBox">
     <property name="text">
      <string>显示接收</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="18" column="13" colspan="2">
    <widget class="QLabel" name="rxBytesLabel">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
    color:red
}</string>
     </property>
     <property name="text">
      <string notr="true">0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="13" colspan="2">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>接收状态</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="28" column="2" rowspan="3" colspan="9">
    <widget class="QTextEdit" name="inputTextEdit">
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="11" colspan="2">
    <widget class="QPushButton" name="clearOutputPushButton">
     <property name="text">
      <string>清空输出</string>
     </property>
    </widget>
   </item>
   <item row="31" column="6">
    <widget class="QCheckBox" name="bigeEndianCheckBox">
     <property name="text">
      <string>大端CRC</string>
     </property>
    </widget>
   </item>
   <item row="27" column="9">
    <widget class="QLabel" name="label_17">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>当前CRC值</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="29" column="1">
    <widget class="QComboBox" name="crcParameterModelsComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="31" column="0">
    <widget class="QPushButton" name="readinFilePushButton">
     <property name="text">
      <string>读入文件</string>
     </property>
    </widget>
   </item>
   <item row="5" column="11" colspan="2">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>输出数据格式</string>
     </property>
    </widget>
   </item>
   <item row="0" column="11" colspan="4">
    <widget class="QLabel" name="label_5">
     <property name="styleSheet">
      <string notr="true">QLabel{
    color:green
}</string>
     </property>
     <property name="text">
      <string>输出设置</string>
     </property>
    </widget>
   </item>
   <item row="16" column="13" colspan="2">
    <widget class="QLabel" name="rxSpeedLabel">
     <property name="styleSheet">
      <string notr="true">QLabel{
    color:red
}</string>
     </property>
     <property name="text">
      <string notr="true">0B/s</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="9">
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">QLabel{
    color:green
}</string>
     </property>
     <property name="text">
      <string>数据输出</string>
     </property>
    </widget>
   </item>
   <item row="9" column="11" colspan="2">
    <widget class="QCheckBox" name="saveOutputFileToFilecheckBox">
     <property name="text">
      <string>写入文件</string>
     </property>
    </widget>
   </item>
   <item row="9" column="13" colspan="2">
    <widget class="QPushButton" name="outputFilePathPushButton">
     <property name="text">
      <string>保存设置</string>
     </property>
    </widget>
   </item>
   <item row="29" column="11" colspan="2">
    <widget class="QPushButton" name="autoResponseSettingPushButton">
     <property name="text">
      <string>自动回复</string>
     </property>
    </widget>
   </item>
   <item row="28" column="0">
    <widget class="QCheckBox" name="cycleEnableCheckBox">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="text">
      <string>定时(ms)</string>
     </property>
    </widget>
   </item>
   <item row="6" column="13" colspan="2">
    <widget class="QCheckBox" name="autoWrapCheckBox">
     <property name="text">
      <string>自动换行</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="13" colspan="2">
    <widget class="QCheckBox" name="showTxDataCheckBox">
     <property name="text">
      <string>显示发送</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="31" column="10">
    <widget class="QPushButton" name="sendPushButton">
     <property name="text">
      <string>发送数据</string>
     </property>
    </widget>
   </item>
   <item row="12" column="11" colspan="4">
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="17" column="13" colspan="2">
    <widget class="QLabel" name="rxFramesLabel">
     <property name="styleSheet">
      <string notr="true">QLabel{
    color:red
}</string>
     </property>
     <property name="text">
      <string notr="true">0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="13" colspan="2">
    <widget class="QCheckBox" name="showMsCheckBox">
     <property name="text">
      <string>显示毫秒</string>
     </property>
    </widget>
   </item>
   <item row="31" column="7">
    <widget class="Line" name="line_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="29" column="13" colspan="2">
    <widget class="QPushButton" name="highlightSettingPushButton">
     <property name="text">
      <string>高亮显示</string>
     </property>
    </widget>
   </item>
   <item row="30" column="13" colspan="2">
    <widget class="QPushButton" name="moreSettingsPushButton">
     <property name="text">
      <string>更多设置</string>
     </property>
    </widget>
   </item>
   <item row="31" column="13" colspan="2">
    <widget class="QPushButton" name="dataVisualizationPushButton">
     <property name="text">
      <string>数据图表</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="19" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">QFrame {
	border:none;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item row="2" column="1">
       <widget class="QPushButton" name="switchPushButton">
        <property name="text">
         <string>打开设备</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>---------------------------------------------------------------------------</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QFrame" name="deviceSettingFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>----------------------------------------------------------------------------------------------------</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="refreshPushButton">
        <property name="text">
         <string>刷新设备</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="21" column="12">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" rowspan="21" colspan="9">
    <widget class="QTextBrowser" name="outputTextBroswer"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clearOutputPushButton</sender>
   <signal>clicked()</signal>
   <receiver>outputTextBroswer</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>674</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
